wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ReservedElement.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2016 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.wire.schema.internal.parser
+
+import com.google.common.collect.Range
+import com.squareup.wire.schema.Location
+import com.squareup.wire.schema.internal.Util.appendDocumentation
+
+data class ReservedElement(
+  val location: Location,
+  val documentation: String = "",
+  /** A [String] name or [Integer] or [Range&amp;lt;Integer&gt;][Range] tag.  */
```

Comment:
are the HTML escapes needed now that we're markdown? Since you're making a link should it be surrounded with `` ` `` so it's not plaintext but code font?

Reply:
The escapes are not needed.  Fixed.